test/videojs-contrib-hls.test.js

```diff
@@ -315,25 +315,17 @@ QUnit.module('HLS', {
 });
 
 QUnit.test('starts playing if autoplay is specified', function() {
-  let plays = 0;
-
   this.player.autoplay(true);
   this.player.src({
     src: 'manifest/playlist.m3u8',
     type: 'application/vnd.apple.mpegurl'
   });
 
-  // REMOVEME workaround https://github.com/videojs/video.js/issues/2326
-  // this.player.tech_.triggerReady();
-  // this.clock.tick(1);
   // make sure play() is called *after* the media source opens
-  this.player.tech_.hls.play = function() {
-    plays++;
-  };
   openMediaSource(this.player, this.clock);
 
   standardXHRResponse(this.requests[0]);
-  QUnit.strictEqual(1, plays, 'play was called');
```

Comment:
I think that some of these unit test changes may have been from an old version of my-rebase, as I did not change any of them.